From 39a814343be0df56f3203320154b15720748df1f Mon Sep 17 00:00:00 2001
From: Jonathan Helmus <jjhelmus@gmail.com>
Date: Wed, 6 Nov 2019 14:13:24 -0600
Subject: [PATCH 1/2] Don't use USE_BCRYPTGENRANDOM for VS older than 2015

---
 crypto/rand/rand_win.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/crypto/rand/rand_win.c b/crypto/rand/rand_win.c
index 9036546008..7ee7870bca 100644
--- a/crypto/rand/rand_win.c
+++ b/crypto/rand/rand_win.c
@@ -19,7 +19,7 @@
 
 # include <windows.h>
 /* On Windows Vista or higher use BCrypt instead of the legacy CryptoAPI */
-# if defined(_MSC_VER) && _MSC_VER > 1500 /* 1500 = Visual Studio 2008 */ \
+# if defined(_MSC_VER) && _MSC_VER >= 19000 /* 19000 = Visual Studio 2015 */ \
      && defined(_WIN32_WINNT) && _WIN32_WINNT >= 0x0600
 #  define USE_BCRYPTGENRANDOM
 # endif
-- 
2.32.0.windows.2

